functions/Connect-DbaInstance.ps1

```diff
@@ -448,7 +448,7 @@ function Connect-DbaInstance {
     }
     process {
         if (Test-FunctionInterrupt) { return }
-        if ($Tenant -and -not $AccessToken) {
+        if ($Tenant -and -not $AccessToken -and $SqlCredential.UserName -match '^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$') {
```

Comment:
It may help to add a comment (block maybe) of one example  or two for what that regex is supposed to catch.

6 months down I seriously doubt we will remember 😆

Reply:
merci, added a comment 😊